pkg/customizations/oscap/oscap.go

```diff
@@ -43,10 +39,25 @@ const (
 	defaultRHEL8Datastream   string = "/usr/share/xml/scap/ssg/content/ssg-rhel8-ds.xml"
 	defaultRHEL9Datastream   string = "/usr/share/xml/scap/ssg/content/ssg-rhel9-ds.xml"
 
-	// tailoring directory path
-	tailoringDirPath string = "/usr/share/xml/osbuild-openscap-data"
+	// oscap related directories
+	DataDir string = "/oscap_data"
 )
 
+type RemediationConfig struct {
+	Datastream         string
+	ProfileID          string
+	TailoringPath      string
+	CompressionEnabled bool
+}
+
+type TailoringConfig struct {
+	RemediationConfig
```

Comment:
I just noticed that embedded the `RemediationConfig` makes the `TailoringConfig` have the tailoring file path set twice. Once in `TailoringPath` and also in `Filepath`. Is there some good reason behind this?

Reply:
Great catch, no I think we can just use the one from the `RemediationConfig`

Reply:
I've added it to the `RemediationConfig` struct since it is potentially needed for both stage options.